.circleci/config.yml

```diff
@@ -145,6 +145,29 @@ jobs:
           name: Build and test with nightly Rust
           command: cargo test --frozen --verbose
 
+  test_integration_nightly:
+    docker:
+      - image: rustlang/rust:nightly
```

Comment:
why on nightly only ?

also why not keeping the build outputs from the build job?